recipes/hidapi/all/conanfile.py

```diff
@@ -0,0 +1,112 @@
+import os
+from conans import ConanFile, AutoToolsBuildEnvironment, MSBuild, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class HidapiConan(ConanFile):
+    name = "hidapi"
+    description = "HIDAPI is a multi-platform library which allows an application to interface " \
+                  "with USB and Bluetooth HID-Class devices on Windows, Linux, FreeBSD, and macOS."
+    topics = ("libusb", "hid-class", "bluetooth")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/libusb/hidapi"
+    license = "GPL-3-or-later", "BSD-3-Clause"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "fPIC": [True, False],
+        "shared": [True, False],
+    }
+    default_options = {
+        "fPIC": True,
+        "shared": False,
+    }
+    generators = "pkg_config"
+    _autotools = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+        if self.settings.compiler == "Visual Studio":
+            self.options.shared = True
```

Comment:
```suggestion
```

Covered by line 37

Reply:
This is about providing a working default option.

Reply:
I see

Reply:
It's probably best to document this..